src/components/results/shortcircuit/shortcircuit-analysis-result-table.tsx

```diff
@@ -355,12 +363,16 @@ const ShortCircuitAnalysisResultTable: FunctionComponent<
         shortCircuitAnalysisStatus,
         !isFetching
     );
-    const rowsToShow = getRows(rows, shortCircuitAnalysisStatus);
+    const rowsToShow = (rows: any) => {
+        let currentRows = getRows(rows, shortCircuitAnalysisStatus);
+        onRowDataUpdated && onRowDataUpdated(currentRows.length === 0);
+        return currentRows;
+    };
```

Comment:
Do not need modify code to invoke onRowDataUpdated, it is a callback method of aggrid

Reply:
done